_includes/pbc/all/install-features/202403.0/install-the-acl-feature.md

```diff
@@ -0,0 +1,436 @@
+use Spryker\Zed\ZedNavigation\ZedNavigationDependencyProvider as SprykerZedNavigationDependencyProvider;
+
+class ZedNavigationDependencyProvider extends SprykerZedNavigationDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ZedNavigationExtension\Dependency\Plugin\NavigationItemCollectionFilterPluginInterface>
+     */
+    protected function getNavigationItemCollectionFilterPlugins(): array
+    {
+        return [
+            new AclNavigationItemCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Installer/InstallerDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\Acl\Communication\Plugin\AclInstallerPlugin;
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins()
+    {
+        return [
+            new AclInstallerPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/User/UserDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\User;
+
+use Spryker\Zed\Acl\Communication\Plugin\GroupPlugin;
+use Spryker\Zed\Kernel\Container;
+use Spryker\Zed\User\UserDependencyProvider as SprykerUserDependencyProvider;
+
+class InstallerDependencyProvider extends SprykerUserDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addGroupPlugin(Container $container)
+    {
+        $container->set(static::PLUGIN_GROUP, function (Container $container) {
+            return new GroupPlugin();
+        });
+
+        return $container;
+    }
+}
+```
+
+**src/Pyz/Zed/Acl/AclDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Acl;
+
+use Spryker\Zed\Acl\AclDependencyProvider as SprykerAclDependencyProvider;
+use Spryker\Zed\AclEntity\Communication\Plugin\Acl\AclEntityAclRolePostSavePlugin;
+use Spryker\Zed\AclEntity\Communication\Plugin\Acl\AclRulesAclRolesExpanderPlugin;
+
+class AclDependencyProvider extends SprykerAclDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\AclExtension\Dependency\Plugin\AclRolesExpanderPluginInterface>
+     */
+    protected function getAclRolesExpanderPlugins(): array
+    {
+        return [
+            new AclRulesAclRolesExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return array<\Spryker\Zed\AclExtension\Dependency\Plugin\AclRolePostSavePluginInterface>
+     */
+    protected function getAclRolePostSavePlugins(): array
+    {
+        return [
+            new AclEntityAclRolePostSavePlugin(),
+        ];
+    }
+}
+```
+
+Use the following example if you want to enable ACL Entity for the whole Application, for example, for the Merchant
```

Comment:
1. What do you mean by "using the example"?
2. What do you mean by `Application`?
3. What do I do if I don't want to enable the ACL Entity?